game/script-topics.rpy

```diff
@@ -599,6 +599,573 @@ label talk_eating_well:
+    # Prompt the player to create an account
+
+    n 1nnmss "'Kay!{w=0.2} Now for step two!"
+    n 1nnmaj "Basically I need something called an API key,{w=0.1} which will let me use that website to find out what the weather is like over there."
+    n 1nlraj "But I can't do that myself...{w=0.3} which is where you come in,{w=0.1} [player]!"
+    n 1nnmss "You'll need to make an account before you can get an API key though."
+    n 1nwrbg "Don't worry{w=0.1} -{w=0.1} it's totally free,{w=0.1} I promise!"
+    n 1nnmss "You can create an account {a=[store.jn_globals.LINK_OPEN_WEATHER_MAP_SIGN_UP]}here{/a},{w=0.1} or you can sign in using the menu at the top."
+    n 1fwdaw "Just make sure to go through all the options carefully{w=0.1} -{w=0.1} don't just dash through it!"
+    n 1nllaj "Oh{w=0.1} -{w=0.1} and make sure you confirm your email address once you've created it,{w=0.1} 'kay?"
+    n 1nnmss "{a=[store.jn_globals.LINK_OPEN_WEATHER_MAP_SIGN_UP]}Here's{/a} that link once more,{w=0.1} just in case!{w=0.2} Just talk to me again when you're done."
+    n 1nwmsm "..."
+    n 1nnmaj "Well,{w=0.1} [player]?{w=0.2} All good?"
+
+    menu:
+        "Yes, I have an account set up.":
+            n 1nchbg "Nice work,{w=0.1} [player]!"
+            n 1nnmss "You'll probably want to make sure you save your login details somewhere secure,{w=0.1} just in case."
+            n 1nnmaj "I hope you didn't forget to confirm your email address too!"
+            n 1nnmss "Alright!{w=0.2} Now here's the challenging part."
+
+        "I already had an account set up.":
+            n 1uwdaj "Oh?{w=0.2} I didn't realise you were such a pro at this,{w=0.1} [player]!{w=0.2} Ehehe."
+
+            if already_discussed_setup:
+                n 1nlrdv "Or maybe we went through this before,{w=0.1} hmm?" #Not quite sure with this expression
+
+            n 1nnmbg "Well, anyway.{w=0.2} The rest of this should be a piece of cake then!"
+
+        "Never mind.":
+            n 1tnmpu "Huh?{w=0.2} You don't wanna continue?"
+            n 1nlraj "That's fine,{w=0.1} I guess."
+            n 1nnmss "Just let me know when you're ready,{w=0.1} 'kay?"
+            return {"lock" : None}
+
+    # Prompt the player for an API key
+
+    n 1nnmbg "Are you ready,{w=0.1} [player]?"
+    n 1nllss "You need to get your API key and send it to me!"
+    n 1nnmbg "You can find your keys {a=[store.jn_globals.LINK_OPEN_WEATHER_MAP_API_KEYS]}here{/a},{w=0.1} or you can get there using the menu like before."
+    n 1nwrts "You should have one set up already,{w=0.1} but you can create another just for me if you want!{w=0.2} Ehehe."
+    n 1nchbg "Don't worry, I won't make you type it all up. You can just paste it,{w=0.2} hehe. "
+    n 1nnmss "{a=[store.jn_globals.LINK_OPEN_WEATHER_MAP_API_KEYS]}Here's{/a} the link in case you forgot{w=0.1} -{w=0.1} and again,{w=0.1} just talk to me when you have it ready!"
+    n 1nnmsm "..."
+    n 1nwdbg "Okaaay!{w=0.2} Take it away,{w=0.1} [player]!"
```

Comment:
I feel like Natsuki's wide eyes here would be better replaced with the closed-happy eyes instead.